README.md

```diff
@@ -1,112 +1,64 @@
-privacy.github.io | ML Privacy &amp; Privacy Eval | A Tutorial at NeurIPS 2024 | Vancouver, British Columbia, Canada
-## Meaningful Privacy-Preserving Machine Learning and How To Evaluate AI Privacy
-In the world of large model development, model details and training data are increasingly closed down, pushing privacy to the forefront of machine learning – how do we protect privacy of the data used to train the model, permitting more widespread data sharing collaborations? How will individuals trust these technologies with their data? How do we verify that the integration of individual’s data is both useful to the rest of the participating federation, and, more importantly - safe for the data owner? How do the regulations integrate into this complex infrastructure?
+# ML Privacy & Privacy Eval Tutorial at NeurIPS 2024
+**Location:** Vancouver, British Columbia, Canada  
+**Date:** December 2024  
+**Website:** [privacy.github.io](https://privacy.github.io)  
 
-These open questions require a multitude of considerations between the incentives of model development,the  data owning parties, and the overseeing agencies. Many cryptographic solutions target these incentives problems, but  are they covering all essential components of trustworthy data sharing? Are they practical, or likely to be practical soon? 
+## **Meaningful Privacy-Preserving Machine Learning and How To Evaluate AI Privacy**
 
-In this tutorial, we attempt to answer questions regarding specific capabilities of privacy technologies in three parts: 1. overarching incentive issues with respect to data and evaluations, 2. Where cryptographic and optimisation solutions can help; for evaluations, we delve deep into secure computation and machine unlearning. 3. Cultural, societal, and research agendas relating to practically implementing these technologies. 
+In large-scale model development, as model details and training data become more restricted, privacy has risen to the forefront of machine learning challenges. How do we safeguard the privacy of data used for model training, enabling broader data-sharing collaborations? How can individuals trust that their data is securely integrated into these models? How do we ensure that the inclusion of data is both beneficial to federated systems and safe for data owners? Moreover, how do regulatory frameworks adapt to support this intricate infrastructure?
```

Comment:
This is a big diff. 

The introduction here sounds too vague for the Neurips audience, who will want very specific wording - we don't talk about regulatory frameworks, and we don't do "secure integration", whatever that is. Safeguard the privacy of data, enabling data sharing are also not the focus.